submissions/rubyAdeniregun/DOM/product.js

```diff
@@ -24,5 +24,79 @@ let submitForm1 = document.getElementById('verifyForm'),
         footeremailvalidation(e);
         return true;  
     }
-    
-    submitForm1.addEventListener("submit", verifyForm);
\ No newline at end of file
+
+submitForm1.addEventListener("submit", verifyForm);
+
+
+//EXAMPLE
+
+let item = document.getElementById("quantityBox")
+    button = document.getElementById("buyNow");
+    shippingCost = document.getElementById("shippingCost");
+    cost = document.getElementById("totalCost");
+    price = document.getElementById("price");
+
+//initial value
+let priceValue = price.innerHTML,
+    costValue = cost.innerHTML;
+//RegEx
+const Regex = /^[1-9]/;
+function calculateQuantity() {
+    if (!item.value.match(Regex)) {
+        button.disable = true
+        button.style = "background-color: grey"
+        item.nextElementSibling.innerHTML = "⚠️Please enter a valid quantity";
+    } else {
+        button.disabled = false;
+        item.nextElementSibling.innerHTML = "";
+    }
+}
+function calculatePrice() {
+    if (item.value >= 1) {
+        let perPrice = "$ " + (2500 * item.value)
```

Comment:
I think `2500` is the default price for one item. Kindly get the value from the HTML element using document.getElement rather than hard coding the price here